codewof/programming/content/en/to-celsius/question.md

```diff
@@ -0,0 +1,7 @@
+# To celsius
+
+Write a function `to_celsius(fahrenheit)` that takes a temperature in fahrenheit `fahrenheit` and **returns** the temperature in celsius. Your function should round the temperature in clesius to 1 decimal place.
+
+For example if we call `to_celsius(68)` the function should return `20`.
+
+*Hint: You will need to use the `round()` function. For example, `round(1.11111, 1)` will result in `1.1`.*
```

Comment:
Regarding the slack convo going on while I'm writing this, I don't think `round` should be used at all